bodhi/server/templates/update.html

```diff
@@ -692,6 +692,23 @@ <h5 class="m-t-2">
                 </div>
               </div>
 
+              % if request.registry.settings.get('ci.required', 'False') == 'True':
```

Comment:
It might be good to throw a .lower() on there so the setting isn't case sensitive.